README.md

```diff
@@ -52,6 +55,7 @@ Ce dépôt comprend de la documentation à l’appui d’[Alerte COVID](https://
 *   [Rapport sur l’accessibilité](https://github.com/cds-snc/covid-alert-documentation/blob/main/RapportAccessibilite.md)
 *   [Processus de divulgation des vulnérabilités](https://github.com/cds-snc/covid-alert-documentation/blob/main/PolitiqueDivulgationVulnerabilites.md)
 
+
```

Comment:
I think here the French might be missing?

Reply:
Could you request changes with the French? I took a guess below 😅

Documentation pour recherche de conception:
*   [Quelle est la réaction des personnes qui ont la COVID-19 face à la « clé à usage unique »?](https://github.com/cds-snc/covid-alert-documentation/blob/main/recherche/revuelitterature.md) (Revue Litterature, Le 28 juillet 2020)